corehq/apps/locations/static/locations/ko/location_types.js

```diff
@@ -80,6 +80,27 @@ hqDefine('locations/ko/location_types.js', function(){
                 }
             });
 
+            // Make sure name and code are unique
+            _.each(['name', 'code'], function (field) {
+                var counts_by_value = _.countBy(self.loc_types(), function (loc_type) {
+                    return loc_type[field]();
+                });
+                var duplicates = [];
+                _.each(counts_by_value, function (count, value) {
+                    if (count > 1) {
+                        duplicates.push(value);
+                        valid = false;
+                    }
+                });
+                _.each(self.loc_types(), function (loc_type) {
+                    var has_error = (field === 'name') ? loc_type.duplicate_name_error : loc_type.duplicate_code_error;
```

Comment:
It is a tiny bit weird that you're iterating over a theoretically generic set of `loc_types` when the error types are only names & codes, but I don't have a better suggestion that wouldn't be overly verbose.